website/docs/r/consul_secret_backend.html.md

```diff
@@ -38,13 +51,21 @@ The following arguments are supported:
   The `namespace` is always relative to the provider's configured [namespace](/docs/providers/vault#namespace).
    *Available only for Vault Enterprise*.
 
-* `token` - (Required) The Consul management token this backend should use to issue new tokens.
+* `token` - (Optional) The Consul management token this backend should use to issue new tokens. This field is required
+when not bootstrapping a new Consul instance.
+
+~> **Important** Because Vault does not support reading the configured token back from the API, Terraform cannot detect
+and correct drift on `token`. Changing the value, however, _will_ overwrite the previously stored values.
+
+* `bootstrap` - (Optional) Denotes that the resource is used to bootstrap the Consul ACL system.
 
-~> **Important** Because Vault does not support reading the configured
-token back from the API, Terraform cannot detect and correct drift
-on `token`. Changing the value, however, _will_ overwrite the previously stored values.
+~> **Important** The bootstrap token is silently stored. If a management token is needed, write a role with the
```

Comment:
May want to start this with  something like  "When  `bootstrap` is true. the token is ...." 

Do you mean  "create another role...." rather than "write a role", not sure someone would know how to do that.

Reply:
Yeah now that I think about it, mentioning the role doesn't make much sense for Terraform since a role resource won't return a token back.